docs/components/clerk-provider.mdx

```diff
@@ -93,14 +93,13 @@ ReactDOM.render(
+| `publishableKey`          | `string`                                                                              | Clerk publishable key for your instance. This can be found in your [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=api-keys).                                                                                                                                                   |
+| `frontendApi`             | `string`                                                                              | The frontend API host for your instance. This can be found in your [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=api-keys) under the **Advanced** section.                                                                                                                                        |
 | `navigate?`               | `(to: string) => Promise<unknown> \| void`                                            | A function which takes the destination path as an argument and performs a "push" navigation.                                                                                                                               |
 | `clerkJSUrl?`             | `string`                                                                              | Define the URL that `@clerk/clerk-react` should hot-load `@clerk/clerk-js` from.                                                                                                                                           |
 | `clerkJSVariant?`         | `string`                                                                              | If your web application only uses Control components, you can set this value to `headless` and load a minimal ClerkJS bundle for optimal page performance.                                                                 |
 | `clerkJSVersion?`         | `string`                                                                              | Define the npm version for `@clerk/clerk-js`.                                                                                                                                                                              |
 | `supportEmail?`           | `string`                                                                              | Optional support email for display in authentication screens. Will only affect [Clerk Components][components-ref] and not [Account Portal][ap-ref] pages.                                                                  |
-| `localization?`           | `object`                                                                              | Optional object to localize your Components. Will only affect Clerk Components and not Account Portal pages.                                                                                                               |

```

Comment:
Is this deprecated or removed?

Reply:
> Is this deprecated or removed?

Were you have issues with this in one of the SDKs?

Reply:
It was listed twice - it's now listed on line 105

Reply:
> It was listed twice - it's now listed on line 105

Awesome, thanks for the update.